src/App.jsx

```diff
@@ -14,70 +16,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
```

Comment:
Try to find 'Jam' another way, if it's not at 8-th index you will not have it on a start position. I used 'find' for this 
  const defaultGood = goods.find(el => el === 'Jam');
  const [selectedGood, setSelectedGood] = useState(defaultGood);